src/collections/_documentation/platforms/python/index.md

```diff
@@ -2,55 +2,373 @@
+- [Full documentation on Environments]({%- link _documentation/enriching-error-data/environments.md -%})
+
+### Breadcrumbs
+
+Sentry will automatically record certain events, such as changes to the URL and XHR requests to provide context to an error.
+
+You can manually add breadcrumbs on other events or disable breadcrumbs.
+
+```python
+from sentry_sdk import add_breadcrumb
+
+add_breadcrumb(
+    category='auth',
+    message='Authenticated user %s' % user.email,
+    level='info',
+)
+```
+
+For more information, see:
+
+- [Full documentation on Breadcrumbs]({%- link _documentation/enriching-error-data/breadcrumbs.md -%})
+- [Debug Issues Faster with Breadcrumbs](https://blog.sentry.io/2016/05/04/breadcrumbs).
+
+### Filter Events & Custom Logic
+
+Sentry exposes a beforeSend callback which can be used to filter out information or add additional context to the event object.
+
+```python
+import sentry_sdk
+
+def strip_sensitive_data(event, hint):
+    # modify event here
+    return event
+
+sentry_sdk.init(
+    before_send=strip_sensitive_data
+)
+```
+
+For more information, see:
+
+- [Full documentation on Filtering Events]({%- link _documentation/error-reporting/configuration/filtering.md -%})
+- [Manage Your Flow of Errors Using Inbound Filters](https://blog.sentry.io/2017/11/27/setting-up-inbound-filters).
+
+### Integrations
+
+*Integrations* extend the functionality of the SDK for some common frameworks and libraries. Similar to plugins, they extend the functionality of the Sentry SDK. A call to `sentry_sdk.init` configures integrations. Unless you set `default_integrations` to `False`, Sentry automatically adds any default integration not in the list below.
+
+#### Web Frameworks
+
+- [Django]({%- link _documentation/platforms/python/django.md -%})
+- [Flask]({%- link _documentation/platforms/python/flask.md -%})
+- [Sanic]({%- link _documentation/platforms/python/sanic.md -%})
+- [Pyramid]({%- link _documentation/platforms/python/pyramid.md -%})
+- [AIOHTTP]({%- link _documentation/platforms/python/aiohttp.md -%})
+- [Tornado]({%- link _documentation/platforms/python/tornado.md -%})
+- [Bottle]({%- link _documentation/platforms/python/bottle.md -%})
+- [Falcon]({%- link _documentation/platforms/python/falcon.md -%})
+- [Generic WSGI]({%- link _documentation/platforms/python/wsgi.md -%})
+
+#### Task Queues
+
+- [Celery]({%- link _documentation/platforms/python/celery.md -%})
+- [RQ (Redis Queue)]({%- link _documentation/platforms/python/rq.md -%})
+
+#### Serverless
+
+- [AWS Lambda]({%- link _documentation/platforms/python/aws_lambda.md -%})
+- [Generic Serverless]({%- link _documentation/platforms/python/serverless.md -%})
+
+#### Other Integrations
+
+- [Logging]({%- link _documentation/platforms/python/logging.md -%})
+- [GNU Backtrace]({%- link _documentation/platforms/python/gnu_backtrace.md -%})
+- [Default integrations]({%- link _documentation/platforms/python/default-integrations.md -%})
+
+### Default Integrations
```

Comment:
If we want this in the index page then please remove `default-integrations.md` and the link in "Other integrations".

I personally would prefer to keep this on its separate page because it's probably something you're only interested in when you need to disable something in the SDK that is supposed to work in the first place.